spec/support/authenticated_session_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module AuthenticatedSessionHelper
+  def use_authenticated_current_user(options = {})
+    klass = options[:klass] || ApplicationController
```

Comment:
I don't see a spec for the ApplicationController that includes auth. I also think it is OK for request specs to be a bit slower, I find them to be analogous to an integration spec if we had views. I think this stubbing fits more with controller specs than with request specs, which wouldn't look very different but do test at a different level